### PR TITLE
[pino] Add symbols types to pino

### PR DESCRIPTION
**Pull request by vlasy**
Relevant pino code: https://github.com/pinojs/pino/blob/db651a51762e8b043f406a73bb19470bcf6dcff1/pino.js#L218

### CODE REVIEW

**Comment by vlasy** [comment]
I would like to add a test but not sure how that should work? Or how to run current tests?

**Comment by AdamVig** [comment]
@vlasy The documentation for tests is [here](https://github.com/DefinitelyTyped/DefinitelyTyped#editing-tests-on-an-existing-package), does that help?

File: types/pino/index.d.ts [hunk 1]
```diff
@@ -46,6 +46,7 @@ declare namespace P {
      */
     const LOG_VERSION: number;
     const levels: LevelMapping;
+    const symbols: { [key: string]: symbol };
```
Comment (AdamVig) [inline types/pino/index.d.ts:49]: This is an improvement, but could we take it a step further and explicitly include the keys for the individual symbols? I copied the key names from [here](https://github.com/pinojs/pino/blob/db651a51762e8b043f406a73bb19470bcf6dcff1/lib/symbols.js#L37-L64).
```suggestion
    const symbols: {
        setLevelSym: symbol,
        getLevelSym: symbol,
        levelValSym: symbol,
        useLevelLabelsSym: symbol,
        mixinSym: symbol,
        lsCacheSym: symbol,
        chindingsSym: symbol,
        parsedChindingsSym: symbol,
        asJsonSym: symbol,
        writeSym: symbol,
        serializersSym: symbol,
        redactFmtSym: symbol,
        timeSym: symbol,
        timeSliceIndexSym: symbol,
        streamSym: symbol,
        stringifySym: symbol,
        stringifiersSym: symbol,
        endSym: symbol,
        formatOptsSym: symbol,
        messageKeySym: symbol,
        nestedKeySym: symbol,
        wildcardFirstSym: symbol,
        needsMetadataGsym: symbol,
        useOnlyCustomLevelsSym: symbol,
        formattersSym: symbol,
        hooksSym: symbol,
    };
```

Comment (vlasy) [reply types/pino/index.d.ts]: Great, thanks :)

**Comment by vlasy** [comment]
> @vlasy The documentation for tests is [here](https://github.com/DefinitelyTyped/DefinitelyTyped#editing-tests-on-an-existing-package), does that help?

OK, I tried adding the symbol access

**Review by AdamVig — APPROVED** [review]
Looks good.

**Comment by vlasy** [comment]
Ready to merge

**Pull request merged**